pylibs/abcdTools/ABCDPlotter.py

```diff
@@ -192,6 +192,9 @@ def calculate_best_points(self):
                   {(mass, ctau): self.contamination_hists[(mass, ctau)]},
                   self.optimization_hists
               )
+    elif self.config.optimization_param == None:
+      warn("Optimization parameter is not set, using 'all' as default")
```

Comment:
This message may be a bit confusing. I think what we want to say is that we'll take parameters from the `abcd_point` dictionary instead of optimizing the method.